source/NoteWindow.cpp

```diff
@@ -77,38 +77,35 @@ const struct tm gettime() {
 }
 
 // Constructor
-NoteWindow::NoteWindow(int32 id)
-		  : BWindow (BRect(100,100,350,350) , "TakeNotes", B_DOCUMENT_WINDOW, B_ASYNCHRONOUS_CONTROLS){
-
-	BString 	title("Untitled Note ");
+NoteWindow::NoteWindow()
+	:
+	BWindow(BRect(100, 100, 350, 350), "TakeNotes", B_DOCUMENT_WINDOW,
```

Comment:
I didn't realize that removing the object `title` and not setting the window title later would require changing the `BWindow` constructor here:
```suggestion
	BWindow(BRect(100, 100, 350, 350), "Untitled", B_DOCUMENT_WINDOW,
```